pkg/services/sqlstore/org.go

```diff
@@ -122,6 +122,13 @@ func CreateOrg(cmd *m.CreateOrgCommand) error {
 func UpdateOrg(cmd *m.UpdateOrgCommand) error {
 	return inTransaction2(func(sess *session) error {
 
+		orgQuery := &m.GetOrgByIdQuery{Id: cmd.OrgId}
+
+		orgDoesntExists := GetOrgById(orgQuery)
```

Comment:
Relaying on affected rows might not work with all databases.
https://golang.org/pkg/database/sql/#Result